@here/harp-mapview/lib/Utils.ts

```diff
@@ -33,6 +33,8 @@ const groundNormalPlanarProj = new THREE.Vector3(0, 0, 1);
 const groundPlane = new THREE.Plane(groundNormalPlanarProj.clone());
 const groundSphere = new THREE.Sphere(undefined, EarthConstants.EQUATORIAL_RADIUS);
 const rayCaster = new THREE.Raycaster();
+const maxTiltAngleAllowed = Math.PI / 2;
```

Comment:
There is already a hardcoded value of 89 in MapView.lookAt. Please move this constant to a central place and reuse it.

Reply:
Ok, saw it, although it is slight different :).